src/base/utils/foreignapps.cpp

```diff
@@ -223,7 +230,12 @@ namespace
         // Fallback: Detect python from default locations
         const QFileInfoList dirs = QDir("C:/").entryInfoList({"Python*"}, QDir::Dirs, (QDir::Name | QDir::Reversed));
         for (const QFileInfo &info : dirs) {
+            const QString path3 {info.absolutePath() + "/python3.exe"};
```

Comment:
I would suggest another name: `py3Path` and `pyPath` for the `path` below (please rename it too).